packages/js/src/general/store/alert-center.js

```diff
@@ -106,6 +119,14 @@ const selectActiveAlerts = createSelector(
 	( alerts ) => alerts.filter( ( alert ) => ! alert.dismissed )
 );
 
+/**
+ * Selector to get the resolve success message.
+ *
+ * @param {object} state The state.
+ * @returns {string} The resolve success message.
+ */
+const selectResolveSuccessMessage = ( state ) => get( state, `${ ALERT_CENTER_NAME }.resolveSuccessMessage`, null );
```

Comment:
I think it's nicer to use array:
```suggestion
const selectResolveSuccessMessage = ( state ) => get( state, [ ALERT_CENTER_NAME , resolveSuccessMessage ], null );
```

Reply:
Hm, I just used the same format with a couple of line below:
```
const selectAlertToggleError = ( state ) => get( state, `${ ALERT_CENTER_NAME }.alertToggleError`, null );
```